core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/TimeData.kt

```diff
@@ -27,6 +29,7 @@ data class TimeData(
      * stop has been reached yet.
      */
     val maxDepartureDelayingWithoutConflict: Double,
+    val maxFirstDepartureDelaying: Double,
```

Comment:
Might be worth adding a comment for this value here, just in case people get confused.

Reply:
I've added a comment there.

I'll probably rename variables and rewrite comments in this class, it's a bit messy. But I'd do it in a PR that has no functional change.